backend/algorithms/objects/categories.py

```diff
@@ -149,3 +149,24 @@ def match_definition(self, course: str) -> bool:
 
     def __str__(self) -> str:
         return self.faculty
+
+
+class GenEdCategory(Category):
+    """
+    Category for a course being a Gened
+        - This is condition is unique in the sense that it's matching is
```

Comment:
Oh god does that mean we need to pass down gen eds?

Reply:
Indeed; Either they get attached to the class or, the `is_allowed` method needs to do a lookup to something else.

An alternative is to pass in the `User` class to `match_definition` and then have some method on the user to list / match against gen-eds